bucket/renpy.json

```
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://www.renpy.org/",
+    "description": "Popular open source visual novel engine that uses Python for scripting",
+    "license": "MIT",
+    "version": "7.3.2",
+    "url": "https://www.renpy.org/dl/7.3.2/renpy-7.3.2-sdk.7z.exe#dl.7z",
+    "hash": "8d6e511ce39324e1ea2d659af727e3bbf7969f10fef1a313241391e2a6ff77a6",
+    "extract_dir": "renpy-7.3.2-sdk",
+    "bin": "renpy.exe",
+    "shortcuts": [
+        [
+            "renpy.exe",
+            "RenPy"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.renpy.org/release_list.html",
+        "regex": "Ren'Py ([\\d\\.]+)"
+    },
+    "autoupdate": {
+        "url": "https://www.renpy.org/dl/$version/renpy-$version-sdk.7z.exe#dl.7z",
+        "hash": {
+            "url": "https://www.renpy.org/dl/$version/checksums.txt",
+            "regex": "$sha256.*?$basename"
```

I think this could be removed. But needs to be tested.

```suggestion
```

---

I just tested it and it works fine but returns the md5 checksum by default. 
If that's okay with you, I'm commiting this.

---

Ok then go with this. No need to use greedy match all when it is only whitespace.

```suggestion
            "regex": "$sha256\\s+$basename"
```